### PR TITLE
Vision p6 add

### PR DESCRIPTION
**Pull request by psharath1**
BUG= Added support for ADD operator on Vision P6

### CODE REVIEW

File: tensorflow/lite/micro/kernels/xtensa/xtensa_add.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```
Comment (rascani) [inline tensorflow/lite/micro/kernels/xtensa/xtensa_add.h:1]: 2022

File: tensorflow/lite/micro/kernels/xtensa/add_vision.cc [hunk 2]
```diff
@@ -0,0 +1,121 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```
Comment (rascani) [inline tensorflow/lite/micro/kernels/xtensa/add_vision.cc:1]: 2022

**Review by rascani — CHANGES_REQUESTED** [review]
LGTM. Just a couple nits on the new file license year.

File: tensorflow/lite/micro/kernels/xtensa/xtensa_add.h (hunk 1 above)
Comment (psharath1) [reply tensorflow/lite/micro/kernels/xtensa/xtensa_add.h]: updated

File: tensorflow/lite/micro/kernels/xtensa/add_vision.cc (hunk 2 above)
Comment (psharath1) [reply tensorflow/lite/micro/kernels/xtensa/add_vision.cc]: updated

**Review by rascani — DISMISSED** [review]

**Review by tingyan19 — DISMISSED** [review]

**Review by tingyan19 — APPROVED** [review]

**Pull request merged**